guides/source/active_record_querying.md

```diff
@@ -922,23 +1005,50 @@ Null Relation
 The `none` method returns a chainable relation with no records. Any subsequent conditions chained to the returned relation will continue generating empty relations. This is useful in scenarios where you need a chainable response to a method or a scope that could return zero results.
 
 ```ruby
-Article.none # returns an empty Relation and fires no queries.
+Order.none # returns an empty Relation and fires no queries.
 ```
 
+This allows you return an empty relation so that calling methods don't have to do a check for nil.
+
 ```ruby
-# The visible_articles method below is expected to return a Relation.
-@articles = current_user.visible_articles.where(name: params[:name])
+# You can use all of the superclass methods dealing with orders without worrying about dealing with nil. 
+class Guest < Customer
 
-def visible_articles
-  case role
-  when 'Country Manager'
-    Article.where(country: country)
-  when 'Reviewer'
-    Article.published
-  when 'Bad User'
-    Article.none # => returning [] or nil breaks the caller code in this case
+  def orders
+    Order.none  # A guest never has any orders. 
   end
 end
+````
+
+```ruby
+  # in the Customer class:
+  # ---------------------- 
+  # The reviews_to_publish method below is expected to return a Relation.
+  # It might be called like this:
+  # @pub_reviews = customer.reviews_to_publish.where(id: params[:id])
```

Comment:
Why is this a code comment? Would it make more sense to just write it in markdown above? Also if this should be in the customer class we don't need to note that, we can just open the class in the example method below.

I would also consider giving the method snippet first and then mention how it is called after.